prow/config/config.yaml

```diff
@@ -84,8 +84,13 @@ branch-protection:
           # Use this to specify that a status coming from outside of prow is
           # required.  We use this to require functional jobs running in
           # jenkins are required, for example.
-          required_status_checks:
-            contexts: ["test-v1b1-integration"]
+          branches:
+            main:
+              required_status_checks:
+                contexts: ["test-v1b1-integration"]
```

Comment:
Are we going to add also the `test-v1b1-centos-integration` for bmo PRs?

Reply:
+1 adding it in the same PR

Reply:
Are we ready yet? Or shall we do this once we have more quota on fra? I dnt have any strong opinion on this. I can add it in the same pr if you folks things its fine

Reply:
Oh wait, we still run integration tests in kna dnt we?

Reply:
yes integration tests are running in kna1. So we shouldn't face any quota issue

Reply:
just did a quick check, and seems:
- master integration tests are running in Fra1
- PR integration tests in Kna1

Reply:
So, yes, feature tests, e2e tests, main, master and release* branch tests all are running in Fra, all the remaining ones in Kna.
https://github.com/metal3-io/project-infra/blob/b65db2070e7674acfbaa0a3cb943a7f1252c905f/jenkins/scripts/integration_test.sh#L53-L59

Reply:
added